src/panoptes/pocs/filterwheel/filterwheel.py

```diff
@@ -212,6 +236,23 @@ def move_to(self, position, blocking=False):
 
         return self._move_event
 
+    def move_to_dark_position(self):
+        """ Move to filterwheel position for taking darks. """
+        if self.dark_position:
+            self.move_to(self.dark_position)
+        else:
+            msg = f"Request to move to dark position but {self} has no dark_position set."
+            self.logger.warning(msg)
+
+    def move_to_light_position(self):
+        """ Return to previous filterwheel position from before taking darks. """
+        if self._light_position:
+            self.move_to(self._light_position)
+        else:
+            msg = f"Request to revert to previous light position but {self} has" + \
+                "no _light_position stored."
```

Comment:
Is `dark_position` public and `_light_position` private for any reason?  If not let's make consistent.

Reply:
I made `dark_position` into a read only property so there is both `dark_position` and `_dark_position`. At the moment though the only code outside the `FilterWheel` class that accesses this is the parent `Camera`, and I can't think of any other reasons why someone else would need to access it so I guess I'll remove that property.